### PR TITLE
Docs: Weekly update for April 21

### PR DESCRIPTION
**Pull request by ponori**

### CODE REVIEW

**Review by AlbertCarreras — APPROVED** [review]

File: docs/pages/BlogPosts.json [hunk 1]
```diff
@@ -2,6 +2,61 @@
...
+            "content": "We've shipped a pilot version of our [SheetMobile component](https://gestalt.pinterest.systems/web/sheetmobile). This represents one of our highest priorities for H1, so we're super excited for this initial launch. We're still working on some kinks, but it's well on its way to being ready to use. We look forward to having more updates on this topic."
+          },
+          {
+            "title": "New DateField component (experimental version)",
+            "audience": ["Design", "Engineering"],
+            "imageColor": "blue-skycicle-0",
+            "imageSrc": "https://i.pinimg.com/originals/d0/eb/c1/d0ebc19d8630dec7bdc47bdda93037ea.png",
+            "imageAltText": "Example of Gestalt's DateField component.",
+            "content": "We shipped an experimental [DateField component](https://gestalt.pinterest.systems/web/datefield) to address an pressing need within product. This component also has some kinks to iron out, but we felt it was ready to ship in an experimental state. This was a great team effort to get a workable solution out the door in such a short time, so big congrats to all involved!"
```
Comment (AlbertCarreras) [inline docs/pages/BlogPosts.json:23]: address **a** pressing

File: yarn.lock [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 "@actions/core@^1.9.1":
   version "1.9.1"
-  resolved "https://registry.yarnpkg.com/@actions/core/-/core-1.9.1.tgz#97c0201b1f9856df4f7c3a375cdcdb0c2a2f750b"
+  resolved "https://registry.npmjs.org/@actions/core/-/core-1.9.1.tgz"
```
Comment (dangerismycat) [inline yarn.lock:7]: There shouldn't be any `yarn.lock` changes for this PR — you'll need to revert this change.

**Review by dangerismycat — CHANGES_REQUESTED** [review]
Screen Shot 2023-04-25 at 6 32 37 PM

This is a weird error. I suspect it may have something to do with the `yarn.lock` changes. I suggest reverting the `yarn.lock` changes and updating the PR. If you still see that error on that CI run, try `rm -rf node_modules && yarn cache clean && yarn`.

File: yarn.lock (hunk 2 above)
Comment (ponori) [reply yarn.lock]: Thanks, that worked!

**Review by dangerismycat — APPROVED** [review]
